agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/Agent.java

```diff
@@ -27,7 +27,7 @@
 /**
```

Comment:
We probably need a better name for this file, `AgentBootstrap`, `AgentDelegate`, etc

Reply:
👍 will do this as part of https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/754#issuecomment-673275367